### PR TITLE
website/developer-docs: add notice about libxmlsec compatibility

### PR DESCRIPTION
**Pull request by BeryJu**
# Details

Add note about libxmlsec compatibility

### New Features

-   Adds feature which does x, y, and z.

### Breaking Changes

-   Adds breaking change which causes \<issue\>.

## Checklist

-   [ ] Local tests pass (`ak test authentik/`)
-   [ ] The code has been formatted (`make lint-fix`)

If an API change has been made

-   [ ] The API schema has been updated (`make gen-build`)

If changes to the frontend have been made

-   [ ] The code has been formatted (`make web`)
-   [ ] The translation files have been updated (`make i18n-extract`)

If applicable

-   [ ] The documentation has been updated
-   [ ] The documentation has been formatted (`make website`)

### CODE REVIEW

File: website/developer-docs/setup/full-dev-environment.md [hunk 1]
```diff
@@ -18,15 +18,19 @@ You can also use a native install, if you prefer.
 
 ## Backend Setup
 
+:::info
+Depending on platform, some native dependencies might be required. On macOS, run `brew install libxmlsec1 libpq`, and for the CLI tools `brew install postgresql redis node@20`
```
Comment (tanberry) [inline website/developer-docs/setup/full-dev-environment.md:22]:
```suggestion
Depending on your platform, some native dependencies might be required. On MacOS, run `brew install libxmlsec1 libpq`, and for the CLI tools `brew install postgresql redis node@20`
```

Comment (BeryJu) [reply website/developer-docs/setup/full-dev-environment.md:22]: macOS is actually stylized lowercase nowadays https://en.wikipedia.org/wiki/MacOS

```suggestion
Depending on your platform, some native dependencies might be required. On macOS, run `brew install libxmlsec1 libpq`, and for the CLI tools `brew install postgresql redis node@20`
```

Comment (tanberry) [reply website/developer-docs/setup/full-dev-environment.md]: Eventually we need to decide either how to handle writing with OS-agnosticism, or... provide a tabbed .mdx section with commands for each OS.

File: website/developer-docs/setup/full-dev-environment.md [hunk 2]
```diff
@@ -18,15 +18,19 @@ You can also use a native install, if you prefer.
...
+:::
+
+:::info
+While [this issue](https://github.com/xmlsec/python-xmlsec/issues/252) is open, a workaround is required to install a compatible version of `libxmlsec1` with brew, see [this comment](https://github.com/xmlsec/python-xmlsec/issues/254#issuecomment-1511135314).
```
Comment (tanberry) [inline website/developer-docs/setup/full-dev-environment.md:26]:
```suggestion
As long as [this issue](https://github.com/xmlsec/python-xmlsec/issues/252) is open, a workaround is required to install a compatible version of `libxmlsec1` with brew, see [this comment](https://github.com/xmlsec/python-xmlsec/issues/254#issuecomment-1511135314).
```

**Review by tanberry — APPROVED** [review]
Thanks so much for adding this caveat! (Now we have to remember to remove it after the issue is fixed.)

**Pull request merged**